.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
+# ignore flake8 checking for __init__.py, because we
+# just want to import something to export them out.
+exclude = *__init__.py*
```

Comment:
Are you running flake8 locally? I'm a little confused about what this is for, b/c I didn't think we had flake8 set up here :-). Also it seems weird that flake8 would need special configuration to handle a common pattern in `__init__.py` files...

Reply:
Oh, I shouldn't include this file :(  It's just because I want to make flake8 happy, because in __init__.py file, we do something like this:
```python
from ._tools import split, catch
```
and split, catch is just import but unused in our `__init__.py` file, flake8 will blame it.  It just says that something is import but unused :(
```